test/test_filesystem.py

```diff
@@ -33,9 +33,9 @@ def test_read_file_missing_file():
 def test_read_file_cannot_read_ioerror(test_file, mocker):
     error = "foobar"
 
-    with mocker.patch("__builtin__.open", side_effect=IOError(error)):
-        with pytest.raises(MoulinetteError) as exception:
-            read_file(str(test_file))
+    mocker.patch("builtins.open", side_effect=IOError(error))
```

Comment:
Apparently if we want to be compatible with both python2 and 3, gotta patch something different depending on the python version number :s

Reply:
Yeah. Do we really want to be compatible with both python2 and 3?

Reply:
I dunno, I was just thinking that would make a smoother transition but if that's too hard we can live without it

Reply:
I tried to keep the compatibility with python 2, the code is ugly however